static_src/main.js

```diff
@@ -121,8 +124,11 @@ function app(orgGuid, spaceGuid, appGuid) {
   activityActions.fetchAppLogs(appGuid);
   quotaActions.fetchAll();
   appActions.changeCurrentApp(appGuid);
-  appActions.fetch(appGuid);
-  appActions.fetchStats(appGuid);
+  appActions.fetch(appGuid).then((res) => {
+    if (appHealth(res) === entityHealth.ok) {
+      appActions.fetchStats(appGuid);
```

Comment:
Comment here would be nice. I'm also curious if a healthy app is the only one where we can fetch stats for.